repondeur/zam_repondeur/templates/visionneuse.html

```diff
@@ -539,11 +539,17 @@ <h3>
     {% for article in articles.values() %}
       {# Once data are cleaner, use the groupby jinja filter here. #}
       {% if loop.previtem is defined and loop.previtem != article or loop.previtem is not defined %}
-        {% if loop.previtem is defined and loop.previtem.titre != article.titre and article.titre != "TODO" or loop.previtem is not defined %}
+        {% if loop.previtem is defined and loop.previtem.titre != article.titre and article.titre not in ("TODO", "") or loop.previtem is not defined and article.titre %}
```

Comment:
😢

Reply:
La condition était déjà trop compliquée pour mon cerveau 😆

Reply:
I'm not giving too much ❤️ to that part because I know it will change soon, using sections titles is not suitable for users.